### PR TITLE
homework3

### PR DESCRIPTION
**Pull request by zFalkRumz**
Sending completed tasks 1-5, tasks 6-7 will be sent soon

### CODE REVIEW

File: Freyak/HW3/PAcrircle.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import math
```
Comment (katemalash) [inline Freyak/HW3/PAcrircle.py:1]: Just advise: it's good practice to add 2 empty lines after imports

File: Freyak/HW3/PAcrircle.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+radius = float(input("Input radius of circle "))
+choise = int(input("Select the action:\nPrint 1 for perimeter\nPrint 2 for area\n"))
+if choise == 1:
+    print("P = ", 2*math.pi*radius)
+if choise == 2:
+    print("S = ", radius**2*math.pi)
+if choise < 1:
```
Comment (katemalash) [inline Freyak/HW3/PAcrircle.py:8]: is it possible to combine this and the next check in the single "if"?

File: Freyak/HW3/PAcrircle.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import math
+radius = float(input("Input radius of circle "))
+choise = int(input("Select the action:\nPrint 1 for perimeter\nPrint 2 for area\n"))
+if choise == 1:
+    print("P = ", 2*math.pi*radius)
+if choise == 2:
+    print("S = ", radius**2*math.pi)
```
Comment (katemalash) [inline Freyak/HW3/PAcrircle.py:7]: '*' has lower priority than '**', so it's recomeded to make spaces around it

File: Freyak/HW3/Pifagor.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+from math import sqrt
```
Comment (katemalash) [inline Freyak/HW3/Pifagor.py:1]: It's recommended to add 2 empty lines after imports

File: Freyak/HW3/Pifagor.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+FirKatet = float(input("Input length of first Katet:  "))
+SecKatet = float(input("Input length of second Katet: "))
+if FirKatet <= 0 or SecKatet <= 0:
+    print("Incorrect input")
+else:
+    print("Hypotenuse = ", sqrt(FirKatet**2+SecKatet**2))
```
Comment (katemalash) [inline Freyak/HW3/Pifagor.py:7]: '+' has lower priority than "*"

File: Freyak/HW3/calc.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
+bytes = float(input("Enter the number of bytes: "))
+choise = float(input("Choose data you want to convert it to:\nPrint 1 fot kBytes\nPrint 2 for mBytes\n "))
+if bytes < 0:
+    print("error")
+if choise == 1:
+    print(bytes / 1000, "Kb")
```
Comment (katemalash) [inline Freyak/HW3/calc.py:6]: are you sure, 1000 is correct value?

File: Freyak/HW3/calc.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+if choise == 2:
+    print(bytes / 1000000, "Mb")
```
Comment (katemalash) [inline Freyak/HW3/calc.py:8]: Are you sure. it's correct value?

**Review by katemalash — CHANGES_REQUESTED** [review]
Please fix Calc task :-)

File: Freyak/HW3/PAcrircle.py (hunk 2 above)
Comment (zFalkRumz) [reply Freyak/HW3/PAcrircle.py:8]: ooops, pushed   incorrect save:)

**Review by katemalash — APPROVED** [review]
Looks good to me

**Pull request merged**